src/src/AdminBundle/Resources/views/autocompleteModal.html.twig

```diff
@@ -12,18 +12,18 @@
                 <div class="modal-body">
                     <div class="form-group row">
                         <div class="col-sm-12">
-                            <label for="birds">{{ modal_content }}</label>
+                            <label for="modal_input_{{ modal_input_name }}">{{ modal_content }}</label>
                             <input
-                                    name="{{ modal_input_name }}"
-                                    data-autocomplete-path="{{ modal_input_path }}"
-                                    id="modal_input_{{ modal_input_name }}"
-                                    type="text" class="form-control"
-                            >
+                                type="text"
+                                class="form-control"
+                                name="{{ modal_input_name }}"
+                                data-autocomplete-path="{{ modal_input_path }}"
+                                id="modal_input_{{ modal_input_name }}"
+                            />
                         </div>
                     </div>
                     <div class="list-group">
-                        <div id="log_{{ modal_input_name }}" class="ui-widget-content log"></div>
-
+                        <div id="log_{{ modal_input_name }}"></div>
```

Comment:
tu triedu by som tam nechala. log kvoli js a to druhe kvoli tomu aby to zobrazilo ako widget nie? otestoval si to bez toho?

Reply:
Otestoval, ta `class`a len rozbijala design, takto je to omnoho krajsie.

Reply:
Oki